packages/extension/cypress/e2e/trustlines.cy.ts

```diff
@@ -287,12 +286,10 @@ const validateTrustlineTx = (destinationAddress: string, currency: string, limit
   cy.get('h1[data-testid="page-title"]').should('have.text', 'Set Trustline');
 
   // Should have the proper information
-  cy.contains('Issuer:').next().should('have.text', destinationAddress);
```

Comment:
Why is there less information now?

Reply:
Because I did not find a nice way to display the issuer on the view. It looked bad imo. But I can put it back, I just need to revert my latest commit

Reply:
I would let it for know, maybe we can think about it in another MR, I think it's important to keep it, though the layout will need an update to have it displayed the right way in the future.
Let's keep it for now :)